recommendations.md

```diff
@@ -127,8 +127,10 @@ Mutual TLS for OAuth 2.0 [@!RFC8705] (see (#pop_tokens)).
 
 ### Refresh Tokens
 
-Refresh tokens MUST be sender-constrained or use refresh token
-rotation as described in (#refresh_token_protection).
+Refresh tokens for public clients MUST be sender-constrained or use refresh
```

Comment:
I would change order of the sentences.

Reply:
I played around with the order of sentences, but I think this order is better to ensure that there is no confusion what is mandated by RFC6749 and what is the new part.